x-pack/plugins/security_solution/public/common/components/landing_page/onboarding/hooks/use_toggle_panel.tsx

```diff
@@ -76,7 +80,9 @@ export const useTogglePanel = ({
 
   const { hash: detailName } = useLocation();
   const stepIdFromHash = detailName.split('#')[1];
+  const spaceId = useSpaceId();
 
+  const OnboardingStorageObject = useMemo(() => new OnboardingStorage(spaceId), [spaceId]);
```

Comment:
Nice solution to keep the storage space-aware 👍
Consider keeping the same name (`onboardingStorage`) for the instance returned, class instances should not start with a capital letter.